bdk-android/lib/build.gradle.kts

```diff
@@ -44,7 +44,7 @@ android {
 }
 
 dependencies {
-    implementation("net.java.dev.jna:jna:5.8.0@aar")
+    implementation("net.java.dev.jna:jna:5.12.0@aar")
```

Comment:
Small inconsistency between files, where some use 5.12 and some use 5.14.

Reply:
thanks! changed.